documentation/src/main/asciidoc/advanced-features.asciidoc

```diff
@@ -19,6 +20,22 @@ SearchFactory searchFactory = fullTextSession.getSearchFactory();
 ----
 ====
 
+[[AccessingSearchIntegrator]]
+=== Accessing the SearchIntegrator
+The SearchIntegrator is an object that gives access to lower level API of Hibernate Search.
+Frameworks can use it to integrate with Hibernate Search. It should not be exposed as a
+public API.
```

Comment:
I don't understand how

> It should **not** be exposed as a public API

can be confused as a recommendation  to the user to now expose it as public API.

I will just remove this sentence anyway, it's related to frameworks and I don't think it's needed here.